deployment/tasks/fixtures.cap

```diff
@@ -1,9 +1,27 @@
 namespace :symfony do
     namespace :mongo do
-        desc 'Load fixtures'
-        task :load_fixtures do
-            on roles(:app) do
-                execute "cd #{fetch(:deploy_to)}/current && php app/console doctrine:mongodb:fixtures:load --env=prod "
+        namespace :all_fixtures do
```

Comment:
in this case all make the command like this: 
mongo:fixtures:all:load
mongo:fixtures:functional:load